catroid/src/main/java/org/catrobat/catroid/ui/ProjectUploadActivity.java

```diff
@@ -148,6 +159,7 @@ public void loadProjectActivity() {
 		verifyUserIdentity();
 	}
 
+	@RequiresApi(api = Build.VERSION_CODES.O)
```

Comment:
ops! our minSdk is 21, please use an alternative code that supports SDKs from 21 on